app/controllers/applications/process_controller.rb

```diff
@@ -123,6 +122,9 @@ def summary
       @application = application
       @result = Views::Applikation::Result.new(application)
       @overview = Views::ApplicationOverview.new(application)
+      @savings = Views::Overview::SavingsAndInvestments.new(application)
+      @benefits = Views::Overview::Benefits.new(application)
+      @income = Views::Overview::Income.new(application)
```

Comment:
I saw that these new assigns are not tested in the controller spec.

Also do we still need the original `@overview` here?